examples/wasi-introduction/wasi-introduction.all.en-us.md

```diff
@@ -0,0 +1,28 @@
+## WASI Introduction
+
+First, we should answer the question, "What is WASI"? WASI is the [WebAssembly System Interface](https://wasi.dev/), "a modular system interface for WebAssembly". To help explain this, let's break it down to:
+
+- **What can WASI do for my WebAssembly Applications?**
+  - WASI offers a [standardized set of APIs](https://github.com/WebAssembly/WASI/blob/master/phases/snapshot/docs.md) for WebAssembly modules to access system resource such as a FileSystem, Networking, Time, Random, etc... .
```

Comment:
My suggestion here would be to mention running wasm "outside the browser" somewhere earlier here, to help new users understand the context here.